src/tool_cb_wrt.c

```diff
@@ -33,7 +33,7 @@
 
 /* create a local file for writing, return TRUE on success */
 bool tool_create_output_file(struct OutStruct *outs,
-                             bool append)
+                             bool allow_overwrite)
```

Comment:
The prototype in `src/tool_cb_wrt.h` needs to be updated to match to avoid confusion.

Reply:
good point!